ES/ESmy023.xml

```diff
@@ -32,99 +32,377 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                                                 <incipit xml:lang="gez">
+                                                                 <locus target="#87va"/>
+                                                                 <hi rend="rubric">፲፪ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ ጸሎቱ፡ የሃሉ፡</hi>
+                                                                   ምስለ፡ አግብርቲሁ፡ አቡነ፡ ገብረ፡ <hi rend="rubric">ዮሐንስ፡</hi> ወለተ፡ <hi rend="rubric">ግብርኤል፡</hi>
+                                                                    ለዓለመ፡ ዓለም፡ አሜን፨ ወሀሎ፡ ፩ብእሲ፡ ዘሥሉጥ፡ ላዕለ፡ ምድረ፡ ኅርጉድ፨ ወሰምዓ፡ ዜና፡ ውእቱ፡ በፅንዓ፡ ኃይሉ፡ እስመ፡
+                                                                    ድኅረ፡ ኵሉ፡ ጸብዕ፡ ወቀትል፡ ይሠራሕ፡ ወአሐተ፡ ዕለተ፡ ሖረ፡ በከመ፡ ልማዱ፡
+                                                                 </incipit>
+                                                                 <note>In other witnesses the name of the land is Ḫərguṣ</note>
+                                                                 </msItem>
+
+                                                                 <msItem xml:id="ms_i2.14">
+                                                                    <locus from="88ra" to="88vb"/>
+                                                                    <title type="complete" ref="LIT6166MiraclesJohnBaptist#Miracle13">Story about the ruler of Bur
+                                                                    who wanted to attack a place with a church of John the Baptist, but was chased away by a strong wind</title>
+                                                                    <incipit xml:lang="gez">
+                                                                    <locus from="88ra" to="88rb"/>
+                                                                    <hi rend="rubric">፲፫።ተአምረሁ፡ ለብፁዕ፡ ወቅዱስ፡ ዮሐንስ፡ መጥምቅ፡ ጸሎቱ፡ ወ</hi>በረከቱ፡ የሃሉ፡ ምስለ፡
+                                                                    አግብርቲሁ፡ አቡነ፡ ገብረ፡ <hi rend="rubric">ዮሐንስ፡</hi>ወለተ፡ <hi rend="rubric">ኢየሱስ፡</hi> ለዓለመ፡
+                                                                    ዓለ፡ አ።<pb n="88rb"></pb> ወካዕበ፡ ስምዑ፡
+                                                                    ንነግረክሙ፡ ዘንተ፡ ተአምረ፡
+                                                                    <supplied reason="omitted" resp="DN"/><pb n="88va"></pb> ተብህለ፡ ከመ፡ ሀሎ፡ ብእሲ፡
+                                                                    መኰንነ፡ ቡር፡ ዘይትመን፡ በጽንዑ፡ ከመ፡ አጋግ፡ አማሌቃዊ፨
+                                                                    ወአሐተ፡ ዕለተ፡ አስተጋብአ፡ ሠራዊቶ፡ ወሖረ፡ ብትዕቢት፡ ወበተዝኅሮ፡ ከመ፡ ይጽብኦሙ፡ ለሰብአ፡
+                                                                    ሀገሩ፡ ለቅዱስ፡ ዮሐንስ፡ መጥምቅ፨ ወሶበ፡ መጽአ፡ ሰምዑ፡ ገብሩ፡ ምህልላ፡ በትጋህ፡
+                                                                    </incipit>
+
+                                                          <colophon xml:id="coloph1" xml:lang="gez">
```

Comment:
the colophon does not belong to the 13th miracle, it belongs to the entire collection Gadl+Miracles or at least to the Miracles. the nesting must be corrected.

Reply:
I renested it.